packages/generate-design-tokens/index.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+
+const DESIGN_TOKENS_FOLDER = path.join(__dirname, '..', '..', '.github', 'mistica-design', 'tokens');
+const SKINS_FOLDER = path.join(__dirname, '..', '..', 'src', 'skins');
+
+const capitalize = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const buildColor = (colorDescription) => {
+    if (colorDescription.value.startsWith('{') && colorDescription.value.endsWith('}')) {
+        return colorDescription.value.replace('{', '').replace('}', '');
+    }
+
+    const colorWithAlphaRegExp = /rgba\(\{(.+)\}, (0\.\d+)\)/;
+    const colorWithAlphaMatches = colorDescription.value.match(colorWithAlphaRegExp);
+    if (colorWithAlphaMatches) {
+        const colorName = colorWithAlphaMatches[1];
+        const alpha = colorWithAlphaMatches[2];
+        return `applyAlpha(${colorName}, ${alpha})`;
+    }
+
+    throw new Error(`Unknown color format: ${colorDescription.value}`);
+};
+
+const buildRadius = (radiusDescription) => {
```

Comment:
Is it safe to assume we will never get something like `1em`?

Reply:
yes, it is. If we need this in the future, we'll need to review this too